lib/ics23/README.md

```diff
@@ -4,86 +4,86 @@ Below an overview of all data types from the [ICS-23 specification](https://gith
 
 ## CommitmentState
 
-spec: `object`  
+spec: `object`\
 impl: _not implemented_
```

Comment:
what does this do?

Reply:
The `\` at the end of the line is a Markdown line break.

It does two things:

1. Ensures a line break after this line
2. Prevents unintended merging with the next line

This is useful for:
- Maintaining specific formatting
- Separating content that shouldn't be combined
- Creating precise line breaks without starting a new paragraph

Without it, some Markdown renderers might combine adjacent lines.
___
in other words, it denotes a hardline break
https://mdformat.readthedocs.io/en/stable/users/style.html#hard-line-breaks
(`mdformat` applied it automatically)